style.css

```diff
@@ -709,24 +769,1231 @@ body {
     color: #7f8cff;
   }
 
-  .social {
-    margin-left: 20%;
-    margin-top: 10px;
+  .grid_container {
+    display: grid;
+    grid-template-columns: (4, 1fr);
+    grid-gap: 10px;
+    width: 90%;
+    height: auto;
+    margin-top: 80px;
+    background-color: #f7f7f9;
   }
 
-  .social img {
-    margin-right: 20px;
+  .grid_desktop {
+    display: grid;
+    grid-column: 1;
+    width: 95%;
+    margin-left: 5%;
+    border-radius: 20px;
+    height: 600px;
+    margin-bottom: 80px;
+    background-image: url(/Asset/arteto.svg);
+    background-size: 50%;
+    background-repeat: no-repeat;
+    background-color: #fff;
   }
 
-  .arrow {
-    display: none;
+  .grid_desktopa {
+    display: grid;
+    grid-column: 1;
+    width: 95%;
+    margin-left: 5%;
+    border-radius: 20px;
+    height: 600px;
+    margin-bottom: 80px;
+    background-image: url(/Asset/proart.svg);
+    background-position: right;
+    background-size: 50%;
+    background-repeat: no-repeat;
+    background-color: #fff;
   }
 
-  .arrow1 {
-    display: none;
+  .grid_desktopb {
+    display: grid;
+    grid-column: 1;
+    width: 95%;
+    margin-left: 5%;
+    border-radius: 20px;
+    height: 600px;
+    margin-bottom: 80px;
+    background-image: url(/Asset/facebook.svg);
+    background-size: 50%;
+    background-repeat: no-repeat;
+    background-color: #fff;
+  }
```

Comment:
- [ ] Since the only property different is the `background-image`, Instead of repeating the same code for `grid_desktop`, `.grid_desktopa`, `grid_desktopb`, and `grid_desktopc`, you can create a class and add it to all elements that need the same rules.